modules/running-distributed-data-science-workloads-disconnected-env.adoc

```diff
@@ -0,0 +1,38 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-disconnected-env_{context}"]
+= Running distributed data science workloads in a disconnected environment
+
+[role='_abstract']
```

Comment:
Regarding this:
"..and you must set some environment variables"

As indicated in the procedure, these environment variables are being set to indicate the location of the Python dependencies.